demo/src/source-info.js

```diff
@@ -177,6 +177,35 @@ const remoteBFTiff = {
   description: 'Tonsil Legacy Bioformats Pyramid Tiff'
 };
 
+// Generated using bioformats2raw and raw2ometiff.
+const remoteTiffRGBUrl =
+  'https://vitessce-demo-data.storage.googleapis.com/test-data/hubmap/test/VAN0008-RK-403-100-PAS_registered.ome.tif';
+
+const remoteTiffRGB = {
+  url: remoteTiffRGBUrl,
+  initialViewState: {
+    zoom: -5,
+    target: [30000, 30000]
+  },
+  dimensions: [
+    {
+      field: 'channel',
+      type: 'nominal',
+      values: ['red', 'green', 'blue']
+    },
+    { field: 'y', type: 'quantitative', values: null },
+    { field: 'x', type: 'quantitative', values: null },
+    { field: 'time', type: 'number', values: null },
+    { field: 'z', type: 'number', values: null }
```

Comment:
The order of dimensions in TIFFs is still so confusing to me. This seems to suggest interleaved, which it is not...

Reply:
This isn't actually used by anything since it's in the OME-TIFF but I can make it a little clearer.